src/systems/material.js

```diff
@@ -264,17 +264,25 @@ function loadImageTexture (src, data) {
 function setTextureProperties (texture, data) {
   var offset = data.offset || {x: 0, y: 0};
   var repeat = data.repeat || {x: 1, y: 1};
+  var npot = data.npot || false;
 
-  // Don't bother setting repeat if it is 1/1. Power-of-two is required to repeat.
-  if (repeat.x === 1 && repeat.y === 1) { return; }
-
-  texture.wrapS = THREE.RepeatWrapping;
-  texture.wrapT = THREE.RepeatWrapping;
-  texture.repeat.set(repeat.x, repeat.y);
+  if (npot) {
+    texture.wrapS = THREE.ClampToEdgeWrapping;
```

Comment:
Because that is a requirement to be able to use NPOT textures directly.  Clamp not wrap/repeat, and no mipmap filtering.